src/common.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
   readStart: '@@@SNIPSTART',
   readEnd: '@@@SNIPEND',
   writeStart: '<!--SNIPSTART',
+  writeStartClose: '-->',
   writeEnd: '<!--SNIPEND',
+  writeEndClose: '-->',
```

Comment:
Is this in use?

Reply:
No, but we probably should use it, now that I think of it or just have a single `writeEnd` defined as `<!--SNIPEND-->`.
WDYT?

Reply:
I'll delete it for now.